### ISSUE TITLE
bug: string.value?

### ISSUE THREAD

**Issue by maquino1985** [graphql-python/graphene-sqlalchemy#225]
https://github.com/graphql-python/graphene-sqlalchemy/blob/db3e9f4c3baad3e62c113d4a9ddd2e3983d324f2/graphene_sqlalchemy/fields.py#L40-L41

AttributeError: 'str' object has no attribute 'value'

**Comment by jnak** [comment]
Thanks for reporting this issue. Can you share more details about the field that caused this error?

**Comment by maquino1985** [comment]
> Thanks for reporting this issue. Can you share more details about the field that caused this error?

Any string value you supply for sort would throw that error because .value is not a valid method on a python string object...

```python
a : str = 'a'
if isinstance(a, str):
    print(a.value)
    
Traceback (most recent call last):
  File "<input>", line 2, in <module>
AttributeError: 'str' object has no attribute 'value'
```

**Comment by maquino1985** [comment]
I just discovered why this code runs, you're really expecting that sort argument to be an instance of `EnumValue`

the if statement works because you define the class as EnumValue(str) but this is really bad imho and I don't understand why you would opt to write the code this way. It should be changed to 'if isinstance(a, EnumValue)'

**Comment by jnak** [comment]
I see. I think what you're proposing makes sense. I don't much about this whole sort functionality so feel free to send a PR. 

Btw I would be in favor to move all the sorting logic into `SQLAlchemyConnectionField` and rename this class to `SortedSQLAlchemyConnectionField`.  And have `UnsortedSQLAlchemyConnectionField` be renamed to `SQLAlchemyConnectionField`. But this is a larger change that will break backward compatibility.

**Comment by erikwrede** [comment]
This seems to be fixed in 3.0:
https://github.com/graphql-python/graphene-sqlalchemy/blob/5da2048f15f16f6e2443a2c2471e9626069dbb04/graphene_sqlalchemy/fields.py#L124
